src/main/webapp/scripts/map.js

```diff
@@ -26,14 +55,39 @@ function initMap() {
   map.addListener("click", e => {
     placeMarkerAndPanTo(e.latLng, map);
   });
+  /*Add Hard-Coded Markers */
+  addMarker(map);
+}
+
+/* Temporary Function to Hold Marker for Sky Garden */
+function addMarker(map) {
+  const skyGarden = {lat: 51.510881, lng: -0.083751}
+  const marker = new google.maps.Marker({
+    position: skyGarden,
+    map: map,
+    title: "Sky Garden"
+  })
+  marker.addListener("click", () => {
+    document.getElementById("rec-container").style.display = "block";
+    map.setZoom(16);
+    map.setCenter(marker.getPosition());
+  });
 }
 
 /* Place Marker Where Map is Clicked On & Show Popup*/
 function placeMarkerAndPanTo(latLng, map) {
+  const greyIcon = {
```

Comment:
nit: just to make it clear, can you please use the [Icon interface](https://developers.google.com/maps/documentation/javascript/reference/marker#Icon) here?

Reply:
I'm not sure what this means, would you be able to elaborate?

Reply:
Sure! Here you are using a [JavaScript object](https://www.w3schools.com/js/js_objects.asp) to represent the icon, but it will be clearer to use the [Icon object](https://developers.google.com/maps/documentation/javascript/reference/marker#Icon) instead. e.g. `new google.maps.Icon(...)`

Reply:
Thanks, I am still a little confused. I read this: https://stackoverflow.com/questions/14407079/what-is-the-construct-interface-google-maps-icon. and I'm not sure which bit of my code to change.

Reply:
Thanks for checking! My bad, looks like it's not an object where you can use the `new` keyword to construct it.

Reply:
no problem! It's a good thing to be keeping in mind!